src/components/BadgeBuilderDialog.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+    <form @submit.prevent="submit">
+        <div ref="modal" class="modal fade" tabindex="-1" data-bs-backdrop="static">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h5 id="exampleModalLabel" class="modal-title">
+                            {{ $t("Generate Badge") }}
+                        </h5>
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close" />
+                    </div>
+                    <div class="modal-body">
+                        <div class="mb-3">
+                            <label for="badge-builder-badge-style" class="form-label">{{ $t("Badge Style") }}</label>
+                            <select id="badge-builder-badge-style" v-model="config.badgeStyle" class="form-select">
+                                <option value="flat">Flat</option>
+                                <option value="flat-square">Flat Square</option>
+                                <option value="for-the-badge">For The Badge</option>
+                                <option value="plastic">Plastic</option>
+                                <option value="social">Social</option>
+                            </select>
+                        </div>
+
+                        <div class="mb-3">
+                            <label for="badge-not-found-text" class="form-label">{{ $t("Return to not found") }}</label>
+                            <input
+                                id="badge-not-found-text" v-model="config.notFoundText" type="text"
+                                class="form-control"
+                            >
+                        </div>
+
```

Comment:
I don't think this is nessesary, please remove it here and in all references to `notFoundText` or `Return to not found`

```suggestion
```